### PR TITLE
Automatic Tax Rate Sorting

### PR DESCRIPTION
**Pull request by mikejolley**
In progress idea to sort tax rates automatically on save to avoid awkward drag and drop UI.

For #10873

### CODE REVIEW

**Comment by mikejolley** [comment]
Think this can be reviewed now.

**Comment by mikejolley** [comment]
Updated with master. @justinshreve 

@daigo75 not sure if you want to take a peek too.

**Comment by daigo75** [comment]
@mikejolley If the sorting takes place during the save action, my changes should be unaffected. I implemented them in such a way that they simulate user's manual input, when the "Save" operation is performed my code has finished doing its job. I will have a look at this as soon as I have some time.

File: assets/css/admin.scss [hunk 1]
```diff
@@ -2042,20 +2042,32 @@ table.wc_input_table {
...
+		padding: 10px;
+	}
+
 	td {
 		padding: 0;
 		border-right: 1px solid #DFDFDF;
 		border-bottom: 1px solid #DFDFDF;
 		border-top: 0;
 		background: #fff;
+		cursor: default;;
```
Comment (justinshreve) [inline assets/css/admin.scss]: Double semicolon

File: assets/js/admin/wc-enhanced-select.js [hunk 2]
```diff
@@ -147,7 +147,7 @@ jQuery( function( $ ) {
 
 				$( this ).select2( select2_args ).addClass( 'enhanced' );
 
-				if( $( this ).data( 'sortable' ) == true ){
+				if( $( this ).data( 'sortable' ) === true ){
```
Comment (justinshreve) [inline assets/js/admin/wc-enhanced-select.js]: There should be a space between `if` and the `parenthesis`. We should also use yoda style here. Ditto with line 230 down below.

File: vendor/autoload.php [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?php
```
Comment (justinshreve) [inline vendor/autoload.php]: Looks like some stuff got added in a merge commit that shouldn't be here. https://github.com/woothemes/woocommerce/pull/11540/commits/c61c08f405fc73a1b8ffa39765a1fdb0a9dc000f

**Comment by justinshreve** [comment]
Left a few notes - just a couple lines needing coding standards fixes. And some extra autoload/vendor files that got added.

**Pull request merged**